google/cloud/bigtable/examples/bigtable_samples_instance_admin_ext.cc

```diff
@@ -63,20 +63,23 @@ void RunInstanceOperations(std::string project_id, int argc, char* argv[]) {
   // [START bigtable_check_instance_exists]
   std::cout << "\nCheck Instance exists:\n";
   auto instances = instance_admin.ListInstances();
+  if (!instances) {
+    throw std::runtime_error(instances.status().message());
+  }
   auto instance_name =
       instance_admin.project_name() + "/instances/" + instance_id.get();
   bool instance_exists =
-      instances.instances.end() !=
+      instances->instances.end() !=
       std::find_if(
-          instances.instances.begin(), instances.instances.end(),
+          instances->instances.begin(), instances->instances.end(),
           [&instance_name](google::bigtable::admin::v2::Instance const& i) {
             return i.name() == instance_name;
           });
-  if (!instances.failed_locations.empty()) {
+  if (!instances->failed_locations.empty()) {
     std::cerr
         << "The service tells us it has no information about these locations:";
-    for (std::string const& location : instances.failed_locations) {
-      std::cout << " " << location << "\n";
+    for (std::string const& location : instances->failed_locations) {
+      std::cout << " " << location;
```

Comment:
The leading sentence before the colon is sent to `std::cerr`, but then each location here is sent to `std::cout`. Is that intentional? Perhaps this one in the loop should also be `std::cerr`?

Note: "continuing anyway" is also sent to `cerr`.